src/api/app/controllers/webui/package_controller.rb

```diff
@@ -786,6 +786,7 @@ def live_build_log
 
     if Project.exists_by_name(params[:project])
       @project = Project.get_by_name(params[:project])
+      @project = @project.name if @project.kind_of?(Project)
```

Comment:
Why do you make a string out of it? Everyone will expect `@project` to be an object.